Python/specialize.c

```diff
@@ -1251,10 +1251,10 @@ static int
 function_kind(PyCodeObject *code) {
     int flags = code->co_flags;
     if ((flags & (CO_VARKEYWORDS | CO_VARARGS)) || code->co_kwonlyargcount) {
-        return SPEC_FAIL_CALL_COMPLEX_PARAMETERS;
+        return SPEC_FAIL_CODE_COMPLEX_PARAMETERS;
```

Comment:
`function_kind` used in `subscr`、`attr` and `call` specialize, move the failure to the common category.